crates/astria-sequencer-utils/src/main.rs

```diff
@@ -0,0 +1,28 @@
+use astria_sequencer_utils::{
+    config::{
+        Command,
+        Config,
+    },
+    genesis_parser::GenesisParser,
+    telemetry,
+};
+use tracing::info;
+
+#[tokio::main]
+async fn main() {
+    let config = Config::get();
+    telemetry::init(std::io::stdout).expect("failed to initialize telemetry");
```

Comment:
Setting up telemetry is probably overkill for a small utility like this. I think just printing to stdout/stderr is sufficient.

Reply:
i removed all the telemetry